c++/src/Statistics.hh

```diff
@@ -1298,9 +1356,17 @@ namespace orc {
       if (_stats.hasMinimum()) {
         tsStats->set_minimumutc(_stats.getMinimum());
         tsStats->set_maximumutc(_stats.getMaximum());
+        if (_minimumNano != DEFAULT_MIN_NANOS) {
+          tsStats->set_minimumnanos(_minimumNano + 1);
```

Comment:
Please add a comment about the reason why we use `+1` here, too.

Reply:
just to be consistent with java side.